Makefile

```diff
@@ -28,7 +28,7 @@ compile:: install-requirements
 install0:: SETUPTOOLS_SCM_PRETEND_VERSION = $(shell python3 -c 'from setuptools_scm import get_version; print(get_version())')
 install0:: compile
 	@echo "==> Installing py-bin/py-deploy into your local virtualenv"
-	/usr/bin/env SETUPTOOLS_SCM_PRETEND_VERSION=$(SETUPTOOLS_SCM_PRETEND_VERSION) pip install ./py-bin
+	/usr/bin/env SETUPTOOLS_SCM_PRETEND_VERSION=$(SETUPTOOLS_SCM_PRETEND_VERSION) pip install -e ./py-bin
 	/usr/bin/env SETUPTOOLS_SCM_PRETEND_VERSION=$(SETUPTOOLS_SCM_PRETEND_VERSION) pip install $(PIP_DEPLOY_OPTIONS) ./py-deploy
```

Comment:
I would install py deploy then also with -e

Reply:
It is using $(PIP_DEPLOY_OPTIONS) = -q -e
I don't know why it was made possible to install it in non-editable mode and editable mode via `make install` and `make install-non-editable`. I can remove that.